src/components/warning-text/_warning-text.scss

```diff
@@ -26,18 +26,19 @@
     top: 50%;
     left: 0;
 
-    min-width: 38px;
-    min-height: 35px;
-    margin-top: -20px; // Half the height of the 38px circle (adjusted for NTA)
+    min-width: 32px;
+    min-height: 29px;
+    margin-top: -20px; // Half the height of the circle (adjusted for NTA)
     padding-top: 3px;
 
+    border: 3px solid govuk-colour("black");
```

Comment:
Would be good to add a comment here to explain why using a border is beneficial – someone who's not thinking about the custom colours use case might well refactor this out.

Reply:
Good shout